OpenRA.Mods.Common/Widgets/Logic/SettingsLogic.cs

```diff
@@ -99,12 +101,13 @@ public SettingsLogic(Widget widget, Action onExit, ModData modData, WorldRendere
 
 				Action closeAndExit = () => { Ui.CloseWindow(); onExit(); };
 				if (current.Sound.Device != OriginalSoundDevice ||
-				    current.Graphics.Mode != OriginalGraphicsMode ||
+					current.Graphics.Mode != OriginalGraphicsMode ||
 					current.Graphics.VideoDisplay != OriginalVideoDisplay ||
-				    current.Graphics.WindowedSize != OriginalGraphicsWindowedSize ||
+					current.Graphics.WindowedSize != OriginalGraphicsWindowedSize ||
 					current.Graphics.FullscreenSize != OriginalGraphicsFullscreenSize ||
-				    current.Graphics.GLProfile != OriginalGLProfile ||
-					current.Server.DiscoverNatDevices != OriginalServerDiscoverNatDevices)
+					current.Graphics.GLProfile != OriginalGLProfile ||
+					current.Server.DiscoverNatDevices != OriginalServerDiscoverNatDevices ||
+					current.Graphics.Language != OriginalLanguage)
```

Comment:
Style nit: I'd prefer to keep the .Graphics entries grouped, so imo this should go between GLProfile and DiscoverNatDevices.